packages/server-wallet/jest/chain-setup.ts

```diff
@@ -1,10 +1,18 @@
 /* eslint-disable no-process-env */
+import * as fs from 'fs';
+
 import {ETHERLIME_ACCOUNTS, GanacheServer} from '@statechannels/devtools';
 import {utils} from 'ethers';
 
 import {deploy} from '../deployment/deploy';
-
+export const ARTIFACTS_DIR = '../../artifacts';
 export default async function setup(): Promise<void> {
+  try {
+    fs.mkdirSync(ARTIFACTS_DIR);
+  } catch (err) {
+    if (err.message !== "EEXIST: file already exists, mkdir '../../artifacts'") throw err;
```

Comment:
Does this prevent multiple, local runs of the chain tests?

Reply:
Nope! It attempts to create the directory and if it fails due to it existing (which would happen when running again) it swallows the error.